static/css/styles.css

```diff
@@ -0,0 +1,63 @@
+/* Profile page */
+
+.profile main {
+    display: block;
+}
+
+.profile h1 {
+    border: 0.3rem solid #00283C;
+    padding: 5rem;
+    text-align: center;
+    grid-area: title;
+}
+
+.profile main section {
+    padding: 1.5rem;
+}
+
+.profile main section h2{
+    text-align: center;
+}
+
+img.profile-pic {
+    display: block;
+    border-radius: 50%;
+    max-width: 20rem;
+    margin: 0.2rem auto;
+}
+.profile ul:first-of-type {
```

Comment:
do you have multiple `ul` within `.profile`?

Reply:
I do.